frontend/src/metabase/ui/components/overlays/Modal/Modal.styled.tsx

```diff
@@ -3,7 +3,7 @@ import { t } from "ttag";
 
 // See `zIndex` prop at https://v6.mantine.dev/core/modal/?t=props
 export const DEFAULT_MODAL_Z_INDEX = 200;
-const DEFAULT_MODAL_SPACING = "lg";
+const DEFAULT_MODAL_SPACING = "xl";
```

Comment:
I'm assuming this was the padding we really wanted, as it's the one I'm seeing everywhere. Before I think we were using lg because it was added to the padding of the content, but we should talk with design before merging